src/Models/EscaperoomRate.php

```diff
@@ -4,19 +4,22 @@
 
 namespace Tipoff\EscapeRoom\Models;
 
+use Tipoff\Support\Contracts\Booking\BookingRateInterface;
 use Tipoff\Support\Models\BaseModel;
 use Tipoff\Support\Traits\HasCreator;
 use Tipoff\Support\Traits\HasPackageFactory;
 use Tipoff\Support\Traits\HasUpdater;
 
-class EscaperoomRate extends BaseModel
+class EscaperoomRate extends BaseModel /*implements BookingRateInterface*/
```

Comment:
@drewroberts @sl0wik I have to do this here, because this class has no relation to satisfy neither `category` nor `getCategory` methods..